.github/CHANGELOG.md

```diff
@@ -64,6 +64,10 @@
 
 <h3>Improvements</h3>
 
+* The `pennylane.plugins` and `pennylane.beta.plugins` folders have been renamed to 
```

Comment:
I would move this into the 'Breaking changes' section, as it is maybe more important to point out!